src/Integration/Spiral/Twig/composer.json

```
@@ -25,10 +25,10 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^8.1",
         "schranz-templating/twig-adapter": "^0.1",
-        "spiral/twig-bridge": "^1.0 || ^2.0",
-        "psr/container": "^1.0 || ^2.0"
+        "spiral/twig-bridge": "^2.0",
+        "spiral/core": "^3.0"
```

Looks like I forget them in this implementation, depending on the use imports I think we need:

```suggestion
        "spiral/boot": "^3.0",
        "spiral/config": "^3.0",
        "spiral/core": "^3.0"
```